com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -231,7 +250,45 @@ public void OnProcessShader(Shader shader, ShaderSnippetData snippetData, IList<
             }
         }
 
-        ShaderFeatures GetSupportedShaderFeatures(UniversalRenderPipelineAsset pipelineAsset)
+        class ShaderBuildPreprocessor : IPreprocessBuildWithReport
```

Comment:
yes it was and I added the code to this branch when adding the stripping functionality without using the code which contained the bug. But I can see this wasn't the best solution to do this because of the added confusion. A better approach would be to merge it together.